app/Http/Controllers/Carts.php

```diff
@@ -0,0 +1,123 @@
+<?php
+
+namespace App\Http\Controllers;
```

Comment:
bổ sung cho anh cái này
```suggestion
namespace App\Http\Controllers;
```